packages/outbox-execute/.env-sample

```diff
@@ -5,7 +5,7 @@
 DEVNET_PRIVKEY="0x your key here"
 
 # Hosted Aggregator Node (JSON-RPC Endpoint)
-L2RPC="https://rinkeby.arbitrum.io/rpc"
+L2RPC="https://goerli-rollup.arbitrum.io/rpc"
```

Comment:
Since rinkeby is migrated, do we want expect tutorial users to use rinkeby or goerli?

Reply:
I thought we would want them to use Goerli